lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -11,12 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+require 'cgi'
+require 'google/api_client'
+require 'google/api_client/auth/compute_service_account'
+require 'googleauth'
+require 'json'
+require 'open-uri'
+require 'socket'
+require 'yaml'
 
 module Fluent
   # fluentd output plugin for the Google Cloud Logging API
   class GoogleCloudOutput < BufferedOutput
     Fluent::Plugin.register_output('google_cloud', self)
 
+    PLUGIN_NAME = 'Fluentd Google Cloud Logging plugin'
+    PLUGIN_VERSION = '0.4.17'
```

Comment:
I only bump this when I actually do a gem release so let's leave it at 0.4.15 for now.